Adding documentation for zone support in vSphere

subramanian-neelakantan opened the pull request:
Writing out the Zone support documentation for VCP.

subramanian-neelakantan commented:
/assign @divyenpatel

subramanian-neelakantan commented:
/cc @skarthiksrinivas @sandeeppsunny

divyenpatel commented on documentation/faqs.md:
```diff
@@ -7,7 +7,7 @@ It has been tested on 1000 node Kubernetes cluster so far. Please refer to this
 
 ## Can vSphere Cloud Provider support Kubernetes Cluster spanning across multiple vCenters, ESXi Clusters and Datacenters?
-With Kubernetes version 1.9.x vSphere cloud provider supports Kubernetes cluster spanning across multiple ESXi clusters, vSphere Datacenters and vCenters. All Kubernetes node VMs must have access to shared storage.
+With Kubernetes version 1.9.x vSphere cloud provider supports Kubernetes cluster spanning across multiple ESXi clusters, vSphere Datacenters and vCenters. All Kubernetes node VMs must have access to shared storage. Also see [zone support](/vsphere-storage-for-kubernetes/documentation/zones.html) for using storage that is shared only by node VMs within a zone. Zone support does not work across mul ...
```
Can we make declaration about zone support with another Question?
We might confuse user with this statement here, as from 1.9.0 we started supporting multi vCenter, but zone support is added in 1.14.x ...

subramanian-neelakantan replied:
Alright. I will remove this mention and leave the zone support in a separate faq.

divyenpatel commented on documentation/zones.md:
```diff
@@ -0,0 +1,229 @@
+
+## Mark Zones and Regions in vCenter
+
+Zones and Regsions are marked by creating and applying Tags in vCenter. The Tags can be applied to a Datacenter, Cluster or ESX Host. Tags are inherited from the parent and can be overridden on the child in this hierarchy. So a Tag applied to a Datacenter is inherited by all Clusters and Hosts in that Datacenter, and a Tag applied to a Cluster overrides any Tags that it might inherit from its par ...
```
Link to documentation on how to create tag and category would help user 

https://docs.vmware.com/en/VMware-vSphere/6.7/com.vmware.vsphere.vcenterhost.doc/GUID-E8E854DD-AA97-4E0C-8419-CE84F93C4058.html

subramanian-neelakantan replied:
Good idea. Added this one.

divyenpatel commented on documentation/zones.md:
```diff
@@ -0,0 +1,229 @@
+
+## Mark Zones and Regions in vCenter
+
+Zones and Regsions are marked by creating and applying Tags in vCenter. The Tags can be applied to a Datacenter, Cluster or ESX Host. Tags are inherited from the parent and can be overridden on the child in this hierarchy. So a Tag applied to a Datacenter is inherited by all Clusters and Hosts in that Datacenter, and a Tag applied to a Cluster overrides any Tags that it might inherit from its par ...
```
Regsions => Regions

subramanian-neelakantan replied:
Done.

divyenpatel commented on documentation/zones.md:
```diff
@@ -0,0 +1,229 @@
+
+## Introduction
+
+In the vSphere infrastructure, kubernetes nodes are run as VMs on an ESX host and Volumes are created as VMDK files on a datastore. The Volume is always placed on a datastore that is shared among all the node VMs of the k8s cluster. This is done so that the Volume can be attached to a pod scheduled on any of the node VMs. However, this implies that there should be a shared datastore among the nod ...
```
Can you replace "kubernetes" => "Kubernetes"  all places. also "k8s" => "Kubernetes" or "K8s"

subramanian-neelakantan replied:
Done.

divyenpatel commented:
@subramanian-neelakantan I have provided some minor comments. Rest all looks good to me.

subramanian-neelakantan commented:
> @subramanian-neelakantan I have provided some minor comments. Rest all looks good to me.

I have addressed your comment in latest commit https://github.com/vmware/vsphere-storage-for-kubernetes/commit/4e7927163c7e361198e126d87d2d8a4a6508c5cd. Kindly take another look.

divyenpatel commented:
/lgtm
